lib/config.py

```diff
@@ -72,6 +72,12 @@
     ('--mock-args',):
         dict(help='Arguments passed to mock command', default=''),
 }
+PUNGI_ARGS = {
+    ('--pungi-binary',):
+        dict(help='Pungi binary path', default='/usr/bin/pungi'),
+    ('--pungi-args',):
+        dict(help='Arguments passed to pungi command', default=''),
```

Comment:
Those defaults will never be used, since the `config.yaml` file takes precedence. You may remove them. This is done for other options in Alan's PR.